lsp/java/src/main/java/com/itsaky/lsp/java/actions/common/ImportOrdererAction.java

```diff
@@ -27,25 +34,57 @@ protected int getTitleTextRes() {
   }
 
   @Override
-  public Boolean execAction(ActionData data) {
+  public void prepare(ActionData data) {
+    super.prepare(data);
+    if (getVisible()) {
+      if (!hasRequiredData(data)) {
```

Comment:
This condition will always evaluate to `false` because you didn't tell `hasRequiredData` what data it should check for. It should be something like `!hasRequiredData (data, CodeEditor.class)`.